docker-compose.test.debug.yaml

```diff
@@ -3,4 +3,4 @@ services:
   ffc-doc-statement-data:
     command: npm run test:debug
     ports:
-      - "9252:9229"
+      - "9282:9229"
```

Comment:
If we are changing test ports away from the default, can we update the ports doc as we go please.
https://eaflood.atlassian.net/wiki/spaces/SFI/pages/4592271438/In+use+ports+for+local+development

I will try and move this to the wiki after we have migrated the service as part of the developer setup update, but for now any port changes can be updated in the existing document so when a new service is added or updated, we don't get clashes.

Reply:
Many thanks, file updated.